recipes-core/sysvinit/sysvinit-inittab/start_getty

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
```

Comment:
Can you mention the commit that deleted the file in the commit message? I'm curious to know why we deleted it before and why you are bringing it back now.

Reply:
Its https://github.com/ni/meta-nilrt/commit/f772bc9bad1ac9a49fa3f9cb22baab2ff011fedb. Added it to commit message. The testing at that time didn't uncover this issue and the code was thought to be unnecessary I believe.